source/extensions/omni.isaac.orbit/omni/isaac/orbit/devices/gamepad/se2_gamepad.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES, ETH Zurich, and University of Toronto
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Gamepad controller for SE(2) control."""
+
+
+import numpy as np
+from typing import Callable
+
+import carb
+import omni
+
+from ..device_base import DeviceBase
+
+
+class Se2Gamepad(DeviceBase):
+    r"""A gamepad controller for sending SE(2) commands as velocity commands.
+
+    This class is designed to provide a gamepad controller for mobile base (such as quadrupeds).
+    It uses the Omniverse gamepad interface to listen to gamepad events and map them to robot's
+    task-space commands.
+
+    The command comprises of the base linear and angular velocity: :math:`(v_x, v_y, \omega_z)`.
+    TODO
```

Comment:
What is this TODO for?

Reply:
Thanks for pointing this out. I wanted to remind me to update the doc string and I removed it in the new commit.